shared/commonprops/commonProps.ts

```diff
@@ -43,6 +44,23 @@ export async function downloadModel() {
   }
 }
 
+export async function updateInjiProps() {
+  try {
+    const response = await Storage.getItem(COMMON_PROPS_KEY);
+    if (response) {
+      const injiProps = JSON.parse(response);
+      injiProps.faceSdkModelUrl = HOST + '/inji';
+      injiProps.credentialRegistry = HOST;
+      injiProps.warningDomainName = HOST;
```

Comment:
if we are using HOST as the main url for three, Saving Inji Props to storage and retrieving them once app started seems like unnecessary operation. As this is for draft implementation, It is fine. but We should refactor this before releasing